way_to_home/tests/unittests/user_profile/test_models.py

```diff
@@ -0,0 +1,106 @@
+"""
+UserProfile Models Test
+========================
+This module provides complete testing for all UserProfile models functions.
+"""
+
+from django.test import TestCase
+from custom_user.models import CustomUser
+from user_profile.models import UserProfile
+
+
+class UserProfileTestCase(TestCase):
```

Comment:
```suggestion
class UserProfileTestCase(TestCase):
```
It is better to change to `UserProfileModelTestCase`.

Reply:
Done!